fmt/format.cc

```diff
@@ -137,11 +147,13 @@ int safe_strerror(
     // A noop assignment operator to avoid bogus warnings.
     void operator=(const StrError &) {}
 
+#if !defined(UNDER_CE)
```

Comment:
If the problem is lack of `errno` then I think an easier workaround is to define a dummy `errno` as a member of `StrError` itself. Then you'll only need one ifdef.

Reply:
The problem is the lack of `errno` and `strerror()`.
So I can add a dummy `errno` member (and I must initialize it in the constructor) and a dummy `strerror()` implementation.

Do you suggest to proceed in this way?

Reply:
I guess adding `errno` won't help then.